tools/template/README.md

```diff
@@ -0,0 +1,43 @@
+# wtstats template
+
+This subfolder contains the code to generate the `wtstats.html.template` file needed by the Python script `wtstats.py`, which is located in the `./tools` directory. 
+
+The template is generated as a node.js "single page" application using several open source frameworks, see Dependencies below. The `wtstats.py` Python script parses stats files, transforms the data and inserts it as JSON into the template. The resulting HTML page can be openend in the browser.
+
+### Build Process
+
+To build the template, follow these steps: 
+
+1. change into the `./tools/template` directory
+2. run `npm install` to install all missing dependencies. 
```

Comment:
Thanks Don!  I downloaded and installed it on the AWS box and looked at the new charts.  @rueckstiess these are very nice charts.  I like the ability to see a log scale with one click and the settings on the left are nice.  Here are a few of my observations:
1.  As noted above, it would be nice if we could self-contain this so it works out of the box rather than having a 2 step download/install of npm itself and then its packages.
2.  When I opened the HTML file in my browser this is all I saw and it wasn't obvious what I should be seeing or that my browser wasn't blocking something or if something was wrong.  Is there some way to default stats being displayed, or at least the left column of individual stats expanded?  Yes, it is easy now that I know what I'm looking at, but I was confused for a few moments as the first-time experience.
![wtstat](https://cloud.githubusercontent.com/assets/2945957/5649650/8b5830c8-9665-11e4-8fb5-c6ffe2df4d5e.jpg)
3.  Once I understood how to use it and played with it, one difference from nvd3 that I missed was the ability to double click on a single item and have it turn off all others.  I.e. in wtstats.log.html, I have say 20 of 24 items turned on (I turned them all on and turned off a few).  Then I really want to see just one or two.  I tried to double-click the one item I wanted.  Instead I had to either turn off the other 18 items one at a time, or turn everything off at the group label and add in the one or two I wanted.  Just wondering if I'm missing a way to do that?